Documentation/project-docs/performance-tests.md

```diff
@@ -12,10 +12,10 @@ Performance test files (if present) are stored within a library's ```tests/Perfo
 **Step # 2:** Change directory to the performance tests directory: ```cd path/to/library/tests/Performance```
 
 **Step # 3:** Build and run the tests:
- - Windows (using admin command shell): ```msbuild /t:BuildAndTest /p:Performance=true /p:ConfigurationGroup=Release /p:TargetOS=Windows_NT```
```

Comment:
I am curious, why was `/p:Performance=true` removed from the instructions here?

It looks like we need it to actually run the tests. Otherwise, they just build.


**Without:**
```text
E:\GitHub\Fork\corefx\src\System.Memory\tests\Performance>dotnet msbuild /t:BuildAndTest /p:ConfigurationGroup=Release
Microsoft (R) Build Engine version 15.9.19+g938f3292a0 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  PerfRunner -> E:\GitHub\Fork\corefx\bin\AnyOS.AnyCPU.Release\PerfRunner\netstandard\PerfRunner.exe
  System.Memory.Performance.Tests -> E:\GitHub\Fork\corefx\bin\AnyOS.AnyCPU.Release\System.Memory.Performance.Tests\netcoreapp\System.Memory.Performance.Tests.dll
```

**With:**
```text
E:\GitHub\Fork\corefx\src\System.Memory\tests\Performance>dotnet msbuild /t:BuildAndTest /p:ConfigurationGroup=Release /p:Performance=true
Microsoft (R) Build Engine version 15.9.19+g938f3292a0 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  PerfRunner -> E:\GitHub\Fork\corefx\bin\AnyOS.AnyCPU.Release\PerfRunner\netstandard\PerfRunner.exe
  System.Memory.Performance.Tests -> E:\GitHub\Fork\corefx\bin\AnyOS.AnyCPU.Release\System.Memory.Performance.Tests\netcoreapp\System.Memory.Performance.Tests.dll
  Using E:\GitHub\Fork\corefx\bin\testhost\netcoreapp-Windows_NT-Release-x64\ as the test runtime folder.
  Executing in E:\GitHub\Fork\corefx\bin\tests\System.Memory.Performance.Tests\netcoreapp-Windows_NT-Release-x64\
  ----- start 13:11:21.67 ===============  To repro directly: =====================================================
...
results
...
```

Reply:
I removed it from the docs because with that PR I enabled running perf test projects without the /p:Performance=true flag. Unfortunately @weshaggard had to default the performance flag to false when he switched to the new bootstrapping model: https://github.com/dotnet/corefx/commit/66392f577c7852092f668876822b6385bcafbd44#diff-4b2f402501d23abbede6eac0f47783e4R127

I'll update the docs.